stdlib/http/path_encode.go

```diff
@@ -0,0 +1,33 @@
+package http
+
+import (
+	"context"
+	"net/url"
+
+	"github.com/influxdata/flux/interpreter"
+	"github.com/influxdata/flux/runtime"
+	"github.com/influxdata/flux/values"
+)
+
+func init() {
+	runtime.RegisterPackageValue("http", "pathEscape", pathEscapeFunc)
+}
+
+const inputStringArg = "x"
```

Comment:
Is `x` a good argument name? Could we use something more descriptive?

I realize this is a common pattern across a lot of our codebase, but I think we could do ourselves a favor and start to break from that tradition a bit.

Reply:
Makes sense - I think I can rename it to indicate it being an input string.